Remotion/Data/DomainObjects.UnitTests/Factories/BaseConfiguration.cs

```diff
@@ -118,5 +124,29 @@ public FakeDomainObjectsConfiguration GetDomainObjectsConfiguration ()
     {
       return new FakeDomainObjectsConfiguration(_mappingLoaderConfiguration, _storageConfiguration, _queryConfiguration);
     }
+
+    public void DisableConnections ()
+    {
+      foreach (var rdbmsProviderDefinition in _storageConfiguration.StorageProviderDefinitions.OfType<RdbmsProviderDefinition>())
+      {
+        var connectionString = rdbmsProviderDefinition.ConnectionString;
+        if (string.IsNullOrEmpty(connectionString))
```

Comment:
Given the PrivateInboke remains as is, we should just update the IF-structure to drop the `continue`